### DISCUSSION TITLE
Welcome to website-templates Discussions!

### DISCUSSION THREAD

**Discussion by learning-zone** [learning-zone/website-templates#26] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by iconic-in** [comment]
Sir I am new pls help me

**Reply by learning-zone** [reply]
Please take any one template reference and try to create your own. This will give you a good learning experience.

**Reply by iconic-in** [reply]
Pls suggest some good learning methods

**Reply by learning-zone** [reply]
Did you see my above comments?

**Reply by iconic-in** [reply]
yes sir . ok thanku sir

**Reply by salizee** [reply]
Please can you help me with 10 page web template?

**Reply by learning-zone** [reply]
Hi @salizee 

What kind of help do you need?

**Reply by salizee** [reply]
Actually I needed a web template to edit within my preference.

**Comment by MyDataSci** [comment]
What kind of license do these templates have?
